plugins/apitracing/src/lib/FunctionHook.cpp

```diff
@@ -56,24 +57,62 @@ namespace ApiTracing
             }
 
             auto extractedParameters = extractor->extractParameters(event, parameterInformation);
-            logParameterList(extractedParameters);
+
+            Json::Value root;
+            Json::StreamWriterBuilder builder;
+            builder["indentation"] = "";
```

Comment:
According to the json-cpp docs you don't need a builder at all. You can also set the indentation options as root node properties.